ISLE/isleapp.cpp

```diff
@@ -540,6 +545,45 @@ void IsleApp::LoadConfig()
 
 	dictionary* dict = iniparser_load(iniConfig);
 
+	// [library:config]
+	// Load sane defaults if dictionary failed to load
+	if (!dict) {
+		SDL_LogInfo(SDL_LOG_CATEGORY_APPLICATION, "Loading sane defaults");
+		FILE* iniFP = fopen(iniConfig, "w");
```

Comment:
Do we really need `\r\n` on Windows? Binary mode is more reliable.
```suggestion
		FILE* iniFP = fopen(iniConfig, "wb");
```